### DISCUSSION TITLE
Is there a Bug in Datafeeder Module of GeoOrchestra?

### DISCUSSION THREAD

**Discussion by haytastan** [georchestra/docker#329] [Q&A] [answered]
While trying to import (feed) data using datafeeder GUI of geOrhestra (i.e at the secont step: 
https://mydomain/import/f8d06d81-2087-4377-9e68-e7f53ebdfaf6/step/2) I am supposed to "select one or more tags that fit my dataset", but nothing is listed in the textbox as shown below.

 
![bug_of_datafeeder](https://github.com/georchestra/docker/assets/7123870/ad4b9318-1bad-4494-94e1-6869130e9d60)

**Comment by jeanmi151** [comment, marked as answer]
Hello,
If you are using the 23.0 branch , you probably didn't change the thesaurus url in the datafeeder configuration, this file : https://github.com/georchestra/datadir/blob/c939dab1816fe53fb91adaed584512c4fa257e36/datafeeder/frontend-config.json#L26

**Comment by haytastan** [comment]
Thanks for your instant reply.
The problem has been solved.
Regards,
Dr.-Ing. Hayati TASTAN
[https://www.linkedin.com/in/hayatitastan/](https://www.linkedin.com/in/hayatitastan/)
